src/olympia/lib/settings_base.py

```diff
@@ -1928,7 +1928,7 @@ def get_sentry_release():
 
 ADZERK_TIMEOUT = 5  # seconds
 ADZERK_NETWORK_ID = env('ADZERK_NETWORK_ID', default=10521)
-ADZERK_SITE_ID = env('ADZERK_SITE_ID', default=1131244)
+ADZERK_SITE_ID = env('ADZERK_SITE_ID', default=1133496)
```

Comment:
What does this default site id correspond to?

Reply:
local dev

Reply:
once this is live on dev we can delete all the non-dev creatives from https://app.adzerk.com/#!/flight/12435798/edit/ - I set up separate flights for dev, stage and local dev and targetted the different sites at each.